lib/brakeman/checks/check_render.rb

```diff
@@ -94,4 +95,37 @@ def safe_param? exp
       end
     end
   end
-end 
+
+  def renderable? exp
+    call?(exp) and
+      sexp?(exp.target) and
+      constant?(exp.target) and
+      (known_renderable_class?(class_name(exp.target)) or method_defined?(class_name(exp.target), :render_in))
+  end
+
+  def known_renderable_class? start
+    traverse_ancestry(start) do |class_name|
+      return true if class_name == :"ViewComponent::Base"
+    end
+  end
+
+  def method_defined? start, name
```

Comment:
Check out `Tracker#find_method` I just merged from https://github.com/presidentbeef/brakeman/pull/1596 I think it takes care of this.